pallets/runtime/tests/src/storage.rs

```diff
@@ -724,6 +730,7 @@ pub type CorporateActions = corporate_actions::Module<TestStorage>;
 pub type Scheduler = pallet_scheduler::Module<TestStorage>;
 pub type Settlement = pallet_settlement::Module<TestStorage>;
 pub type TestUtils = pallet_test_utils::Module<TestStorage>;
+pub type ExternalAgents = pallet_external_agents::Module<TestStorage>;
```

Comment:
In some places we use EA and others we use ExternalAgents. I prefer ExternalAgents over EA.